public/configs/v1/env.json

```diff
@@ -307,7 +308,8 @@
             "stakingAPR": "https://apybara-proxy-web-testnet.infrastructure-34d.workers.dev/v0/protocols/dydx",
             "faucet": "https://faucet.v4dev.dydx.exchange"
          },
-         "stakingValidators": [],
+         "top100WalletAddresses": [],
```

Comment:
Wondering if we should actually move this to vercel env configs instead of within this json

Reply:
Could be a good candidate for statsig dynamic config

Reply:
Yeah, just any way that leaves hardcoded walletAddresses out of our deployer's repos